src/components/auth/LoginForm.tsx

```diff
@@ -0,0 +1,138 @@
+import { Formik } from "formik";
+import { useState } from "react";
+import { StyleSheet, View } from "react-native";
+import { Button, IconButton, Input, Link, Text, Title } from "@/components/common";
+import { ArrowRight, VisibilityOff, VisibilityOn } from "@/assets/icons";
+import { Colors, Fonts } from "@/styles/theme";
+import { useLogin } from "@/services/login";
+import { Notification } from ".";
+import { dictionaryMessage } from "@/constants/validation";
+
+export function LoginForm() {
+  const [hidePassword, setHidePassword] = useState(true);
+  const {
+    handleFormSubmit,
+    formInitialValues,
+    loginSchema,
+    isLoading,
+    errorMessage
+  } = useLogin();
+
+  function handleShowPassword() {
+    setHidePassword(prev => !prev)
+  }
+
+  return (
+    <Formik
+      initialValues={formInitialValues}
+      validationSchema={loginSchema}
+      onSubmit={handleFormSubmit}
+      validateOnChange={false}
+      validateOnBlur={false}
+    >
+      {({
+        handleChange,
+        handleBlur,
+        handleSubmit,
+        values,
+        errors,
+      }) => (
```

Comment:
this is actually pretty tidy overall, but a big tip would be to use the formik hook! it will help pull a heap of indentation out of our JSX